cmake/Modules/LocalGTest.cmake

```diff
@@ -11,17 +11,17 @@ set(GTEST_MAIN_LIB_FILENAME
 if(EXISTS ${GTEST_INCLUDE_DIRS}/gtest/gtest.h AND EXISTS ${GTEST_LIB_FILENAME} AND EXISTS ${GTEST_MAIN_LIB_FILENAME})
     message(STATUS "libavif(AVIF_GTEST=LOCAL): compiled library found in ext/googletest")
 
-    add_library(GTest::gtest STATIC IMPORTED)
-    set_target_properties(GTest::gtest PROPERTIES IMPORTED_LOCATION "${GTEST_LIB_FILENAME}" AVIF_LOCAL ON)
+    add_library(GTest::GTest STATIC IMPORTED)
```

Comment:
Please mention the change of `GTest::gtest` to `GTest::GTest`,  `GTest::gtest_main` to `GTest::Main`, and `Googletest` to `GTest` in the commit message and explain why the new names work better than the old names.

Reply:
Done